applications/treasury-flow.md

```diff
@@ -0,0 +1,335 @@
+| 2c.     | Add Treasury Form         | The dApp will show a form with relevant fields that will trigger the deployment of a new treasury contract                                                                                                                                    |
+| 2d.     | Contract Configuration    | The dApp will allow authorized users to change contract settings = sign txs to interact with smart contract functions                                                                                                                         |
+
+### Milestone 2 — Additional features
+
+- **Estimated duration:** 1.5 month
+- **FTE:** 1
+- **Costs:** 16,000 USD
+
+|  Number | Deliverable               | Specification                                                                                                                                                                                                                                   |
+| ------: | ------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **0a.** | MIT                       |
+| **0b.** | Documentation             | We will provide both **inline documentation** of the code and a basic **tutorial** in the frontend (/docs) that explains how the smart contract can be deployed manually and how the frontend can be setup (also to deploy the smart contracts) |
+| **0c.** | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the readme, we will describe how to run these tests.                                                                                |
+| **0d.** | Docker                    | We will provide a Dockerfile that sets up a complete frontend + contract environment for compiling, testing and interacting with the treasury flow contract and includes all changes made in Milestone 2                                        |
+|     0e. | Article                   | We will publish an **article** on Medium or X that explains all features of the frontend application and smart contract functionality as well as how it could be used in existing tresuries.                                                    |
+|      1. | User Feedback             | We will prepare a form to collect user feedback and incentivize participation in order to find missing functionality and UX issues                                                                                                              |
+|     2a. | Contract Optimizations    | After the audit, we will implement all proposed changes and focus on optimizing storage and contract upgradability                                                                                                                              |
```

Comment:
Just clarifying that we don't actually have an auditor track (and hence no audits) for this cohort. 
Apologies if this wasn't clear at the time you applied.

Reply:
This use case requires the audit, there's no doubt on that.
However, I believe there are other initiatives in the ecosystem focused on providing ecosystem projects support with audits. Such as:
- https://polkadot.subsquare.io/referenda/797
- https://dotpal.io/ 

With that in mind maybe it would make more sense to explicitly include a bullet point that points to using these ecosystem tools to cover the audit. We might be able to assist with introductions / conversations if needed.

Other than that, I'm quite happy with the proposal.

Reply:
I've addressed your feedback. Please take a look and let me know if it looks good.